Makefile

```
@@ -63,7 +63,7 @@ BUILD_FLAGS = -tags osusergo,netgo \
               -ldflags "-s -w -extldflags=-static -X sigs.k8s.io/node-feature-discovery/pkg/version.version=$(VERSION) -X sigs.k8s.io/node-feature-discovery/pkg/utils/hostpath.pathPrefix=$(HOSTMOUNT_PREFIX)"
 
 # multi-arch build with buildx
-IMAGE_ALL_PLATFORMS ?= linux/amd64,linux/arm64
+IMAGE_ALL_PLATFORMS ?= linux/amd64,linux/arm64,linux/arm/v7
```

```suggestion
IMAGE_ALL_PLATFORMS ?= linux/amd64,linux/arm64,linux/arm32v7/v7
```

As seen at https://docs.docker.com/build/building/multi-platform/

---

the link show it with linux/arm/v7

e.g. ` --platform linux/amd64,linux/arm64,linux/arm/v7`

and my local system shows it the same when running `docker buildx ls`.

```
# docker buildx ls
NAME/NODE     DRIVER/ENDPOINT   STATUS    BUILDKIT   PLATFORMS
default*      docker                                 
 \_ default    \_ default       running   v0.13.1    linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/amd64/v4, linux/386, linux/arm64, linux/riscv64, linux/ppc64, linux/ppc64le, linux/s390x, linux/mips64le, linux/mips64, linux/loong64, linux/arm/v7, linux/arm/v6
```